client/dashboard/emails/choose-domain/styles.css

```diff
@@ -14,8 +14,6 @@
 /* Chevron icon color */
 .choose-domain__icon {
 	color: #757575;
-	fill: #757575; /* fallback for SVGs not using currentColor */
```

Comment:
Removing these as the SVG uses `fill: currentColor`.
And `stroke` makes the SVG looks bold.